draft-ietf-rats-architecture.md

```diff
@@ -1304,6 +1306,39 @@ considered.
 The security of conveyed information may be applied at different layers, whether by a conveyance protocol, or an information encoding format. This architecture expects attestation messages (i.e., Evidence, Attestation Results, Endorsements, Reference Values, and Policies) are end-to-end protected based on the role interaction context.
 For example, if an Attester produces Evidence that is relayed through some other entity that doesn't implement the Attester or the intended Verifier roles, then the relaying entity should not expect to have access to the Evidence.
 
+## Handle-based Attestation {#handles-sec}
+
+Handles, described in {{handles}}, can be tampered with, dropped, delayed and
+reordered by an attacker.
+
+An attacker could be either external or belong to the distribution group, for
+example if one of the Attester entities have been compromised.
+
+An attacker who is able to tamper with handles can potentially lock all the
+participants in a certain epoch of choice for ever, effectively freezing time.
+This is problematic since it destroys the ability to ascertain freshness of
+Evidence and Attestation Results.
+
+To mitigate this threat, the transport should be at least integrity protected
+and provide origin authentication.
+
+Dropping handles is equivalent to pinning the victim node to a past epoch, and
+will typically result in a denial of service due to the permanent staleness of
+the Attestation Result or Evidence.
```

Comment:
This statement is not obviously true.   if everyone is locked to a past epoch, then the receiver will continue to accept past handles and it won't be considered stale.  Not sure what was meant.